pandas/tests/frame/indexing/test_get_value.py

```diff
@@ -20,3 +22,16 @@ def test_get_value(self, float_frame):
                 result = float_frame._get_value(idx, col)
                 expected = float_frame[col][idx]
                 assert result == expected
+
+
+def test_enum_value():
```

Comment:
1. Could you name this test `test_enum_column_equality`
2. Could you move this to `pandas/tests/frame/test_arithmetic.py`?
3. Also below could you structure the test like
```
result = q1[Cols.col1] == q2[Cols.col1]
expected = ...
tm.assert_equal(result, expected)
```

Reply:
Sure! Thank you for your guide.

Reply:
@mroeschke do I remove the `.all()` method as well?

Reply:
Correct

Reply:
The changes in this file can be removed now.

Reply:
Sure. About removing `a.all()` when I create a Pandas.Series with the same truth values to compare with the _result_, it tells me that the truth value of a series is ambiguous and I should either of `a.empty, a.bool(), a.item(), a.any() or a.all().`

Reply:
If you inspect `result`, it's a Series

```
In [3]: result
Out[3]:
0    True
1    True
2    True
Name: Cols.col1, dtype: bool
```

So `expected` should also be a `Series`

Reply:
Right. but when I created the series and tried to compare, 

```
data = np.array([True, True, True])
expected = pd.Series(data)
```
checking if `expected == result` gave me the Error that the truth value of the Series is ambiguous, that is why. Unless `tm.assert_equal` accounts for that.

Reply:
Yup, `tm.assert_equal` will check the values and the aspects of the Series

```
In [1]:  from enum import Enum

In [2]:     Cols = Enum("Cols", "col1 col2")
   ...:
   ...:     q1 = DataFrame({Cols.col1: [1, 2, 3]})
   ...:     q2 = DataFrame({Cols.col1: [1, 2, 3]})
   ...:
   ...:     result = q1[Cols.col1] == q2[Cols.col1]

In [3]: result
Out[3]:
0    True
1    True
2    True
Name: Cols.col1, dtype: bool

In [4]: import pandas._testing as tm

In [5]: epxected = pd.Series([True, True, True], name=Cols.col1)

In [6]: tm.assert_series_equal(result, epxected)
```

Reply:
Update: changing it to `tm.assert_series_equal(result, expected)` , hoping it works.

Reply:
Oh, I've just seen your reply. There was a syntax issue that I fixed. `pd.Series` now replaced with `Series` which hopefully should be working now.